### DISCUSSION TITLE
how to remove the previously added automatic code signing for IOS and how to add a new code signing details.

### DISCUSSION THREAD

**Discussion by ritiksonmali** [codemagic-ci-cd/codemagic-docs#2403] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

i want to change the IOS automatic code signing details so how can i change it ?

### Expected results

i want to change the IOS automatic code signing details so how can i change it ?

### Actual results

i want to change the IOS automatic code signing details so how can i change it ?

**Comment by dtrdic** [comment]
Hello @ritiksonmali can you please give me a little bit more information, or maybe a screenshot of the error message you are encountering? Are you looking to change to a different AppStore Connect API key and bundle ID, but you can't?
<img width="1010" alt="Screenshot 2024-03-04 at 15 51 13" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/480ed436-3fd2-4aad-b09d-c50f987fdeaf">

**Reply by dtrdic** [reply]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion
